console/src/components/Common/utils/v1QueryUtils.ts

```diff
@@ -98,6 +98,71 @@ export const getDropPermissionQuery = (
   };
 };
 
+export const getInsertUpQuery = (
+  tableDef: TableDefinition,
+  insertion: object
+) => {
+  const queryCols = Object.keys(insertion);
+  const colString = queryCols.join(',');
+
+  const queryValues = Object.values(insertion);
+  const modifiedValues = queryValues.map(value => {
+    // currently, only modifying for string data,
```

Comment:
the object that this function works with is the one that is present in the response of the `query` call for insertion. and so apart from strings, I'd guess no other data type would require any modification

Reply:
Man! Just use an RQL query :P